packages/automerge-repo-react-hooks/src/useHandles.ts

```diff
@@ -0,0 +1,67 @@
+import { AutomergeUrl, DocHandle, DocumentId } from "@automerge/automerge-repo"
+import { useRef, useState } from "react"
+import { useRepo } from "./useRepo.js"
+
+export interface DocUrlMap<T> {
+  [url: AutomergeUrl]: T
+}
+
+/** useDocuments(docUrls)
+ * maintains a map of document states from { [docUrl]: Doc<T> }
+ * useful for lists and other places you might have a set of documents
+ * does not include change capability at the moment
+ * if you want that, consider adapting this to useHandles and send a PR
+ */
+export function useDocuments<T>(docUrls?: AutomergeUrl[]): DocUrlMap<T> {
```

Comment:
Note: let's rename the file name to `useDocuments` (currently it is `useHandles`).